eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -202,6 +212,9 @@ class VolumePreparer:
             return self.__fail(self.EXIT_INVALID_ARGUMENTS,
                                'Invalid flatpak collection–refs: %s' % refs_list)
 
+        # Add the flatpaks that will be installed by eos-updater-flatpak-installer
+        self._add_autoinstall_flatpaks(flatpak_collection_refs)
```

Comment:
Need to handle exceptions from this call, so that the error message can be formatted nicely and a suitable exit status (`self.EXIT_FAILED`) returned.